New intro & terminology

philarcher opened the pull request:
New intro section written. Markus's terminology added with link to issue/PR commentary.


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/philarcher/did-use-cases/pull/36.html" title="Last updated on Nov 12, 2019, 5:21 PM UTC (909077c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/36/a35d5f5...philarcher:909077c.html" title="Last updated on Nov 12, 2019, 5:21 PM UTC (909077c)">Diff</a>

philarcher committed:
New intro & terminology

New intro section written. Markus's terminology added with link to issue/PR commentary.

jandrieu commented on index.html:
```diff
@@ -73,32 +82,39 @@
-        </p>-->
+      <h2>Introduction</h2>
+      <p>The need for globally unique identifier schemes has been addressed many times. Globally unique ID schemes typically rely on a central authority controlling a 'root' space that is then delegated to local organizations who in turn delegate to further organizations who eventually add the final string to complete the identifer. Even if we restrict ourseleves to online identifiers, there are  ...
+  <li>IANA controls the root namespace of the Internet's Domain Name System; registras operate the top level domains and then license specific domain names to their clients, and it's these clients who create the actual identifiers for online resources (URLs). The example below shows who controls what in a typical URL.
```
registras => registrars

jandrieu commented on index.html:
```diff
@@ -107,30 +123,63 @@ <h2 id="concepts">Concepts of Decentralized Identity</h2>
-  <dd>A document that contains metadata about the <abbr title="Decentralized Identifier">DID</abbr></dd>
+<dl class="termlist">
+<div class="issue" data-number="12">
+  <p>This section is very similar to that proposed in <a href="https://github.com/w3c/did-use-cases/pull/33">PR 33</a>, however, there are comments on that PR that need to be resolved.</p></div>
```
We need newlines here to be able to read it in the diff.

jandrieu commented:
The text should have newlines at somewhere less than 80 characters so we can read it. Many lines just scroll off to the right in the Github editor.

philarcher committed:
Updating focal use cases

These edits address comments from Joe in my previous PR and suggest minor edits to the focal use cases. These edits keep the doc within the scope of the WG and the role of a UCR.

philarcher committed:
Merge branch 'master' into master
